src/style.css

```diff
@@ -1,3 +1,63 @@
+:root {
+  --blue-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  text-transform: uppercase;
+}
+
+.menu {
+  margin-top: 0;
+  margin-right: 0;
+  background-color: white;
+  padding: 0;
```

Comment:
padding should be `0 50px` to add spaces from left and right side as it is on figma